server.js

```diff
@@ -21,7 +22,47 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Hello Book Entusiats!");
```

Comment:
Nice greeting! Another thing that could be added here is some information about what routes to type - like an overview! The routes can alternatively be added in your README file together with your own thoughts.